src/node-lib/curriculum-api-2023/queries/pupilLessonOverview/pupilLessonOverview.gql

```diff
@@ -23,11 +23,10 @@ query pupilLessonOverview(
     supervisionLevel
     contentGuidance
     lessonEquipmentAndResources
-    worksheetUrl
```

Comment:
I've been told that `worksheetUrl` points to the original worksheet (Google Slides) which is only good for embedding.